cli/exp_tasklist.go

```diff
@@ -0,0 +1,142 @@
+package cli
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"golang.org/x/xerrors"
+
+	"github.com/coder/coder/v2/cli/cliui"
+	"github.com/coder/coder/v2/codersdk"
+	"github.com/coder/serpent"
+)
+
+type taskListRow struct {
+	Task codersdk.Task `table:"t,recursive_inline"`
+
+	StateChanged string `table:"state changed"`
```

Comment:
Nit: suggest naming `StateChangedAt` or similar so that the semantics are clearer